integration_tests/pages/match/occupancyViewPage.ts

```diff
@@ -32,7 +31,7 @@ export default class OccupancyViewPage extends Page {
     totalCapacity: number,
     startDate: string,
     durationDays: number,
-    placementRequest: PlacementRequest,
+    placementRequest: PlacementRequestDetail,
```

Comment:
thanks for looking at this @bobmeredith was half expecting this to be an after xmas review. Appreciate it :)

So, in this test code, this function goes on to call the `occupancyViewSummaryListForMatchingDetails()` function to assert some expectations. To achieve this it needs to pass the correct type - which has changed because of the data-type change late on in the PR:
https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/2263/files#diff-cc8326f12e1aed4a0d54a6c3199c6dac2af0450fec3103000188aa31f6df42cb

The justification for that data type change is that, in actual fact, we have always been passing an object of type `PlacementRequestDetail` into that function but the data type was set incorrectly to it's parent type (i.e. `PlacementRequest`). It wasn't flagging issues (as we were passing a child of that parent) but the change is now needed so I can access the application from the `PlacementRequestDetail` (so i can go on to grab the `licenceExpiryDate` later).

Furthermore, in the occupancy view controller, we make the below call:

<img width="2048" alt="placement-request-detail-call" src="https://github.com/user-attachments/assets/c99df356-28f3-41a3-abe1-ddcab1ca1ff7" />

As you can see above, from the IDE hint, what we get back has always been of type `PlacementRequestDetail` (not `PlacementRequest`). And back to the test:
<img width="2048" alt="here" src="https://github.com/user-attachments/assets/64849288-ff1b-4ec4-b1f3-146c88415d9a" />

As you can see we were always stubbing `PlacementRequestDetail` (not the parent `PlacementRequestDetail`) which is why there are no side effects of this data type change also